Makefile

```diff
@@ -65,4 +65,5 @@ build: node_modules/.uptodate
 
 node_modules/.uptodate: package.json yarn.lock
 	yarn install
+	yarn playwright install
```

Comment:
This has a low impact when the binaries are already up to date. In my machine it's around 0.6s, which is acceptable if we only run it when `yarn install` is also needed.